.github/scripts/.build.zsh

```diff
@@ -166,21 +166,33 @@ Usage: %B${functrace[1]%:*}%b <option> [<options>]
   typeset -g OBS_DEPS_VERSION
   setup_${host_os}
 
-  local product_name
-  local product_version
-  read -r product_name product_version <<< \
-    "$(jq -r '. | {name, version} | join(" ")' ${buildspec_file})"
+  local product_name=$(jq -r .name ${buildspec_file})
+  local product_version=$(jq -r .version ${buildspec_file})
+  local product_author=$(jq .author ${buildspec_file})
+  local product_email=$(jq .email ${buildspec_file})
```

Comment:
Why change this to invoke jq multiple times? It creates overhead for context switches into sub shells 4 times instead of just once - you can easily extend the jq call to yield all four fields:

`read -r product_name product_version product_author product_email <<< "$(jq -r '. | {name, version, author, email} | join(" ")' ${buildspec_file})"`

Reply:
@cg2121 If you could merge the calls into the single `jq` call as described, this would be ready to merge overall.

Reply:
This doesn't work if there are spaces in the json value.

Reply:
I've figured out way for it to work.